### PR TITLE
Editorial: use associated Document in place of responsible document

### PR DESCRIPTION
**Pull request by miketaylr**
cf. https://github.com/whatwg/html/pull/7694

***
<a href="https://pr-preview.s3.amazonaws.com/miketaylr/web-locks/pull/109.html" title="Last updated on Mar 12, 2022, 12:20 AM UTC (da1c7ed)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/web-locks/109/7ca557e...miketaylr:da1c7ed.html" title="Last updated on Mar 12, 2022, 12:20 AM UTC (da1c7ed)">Diff</a>

### CODE REVIEW

**Review by saschanaz — APPROVED** [review]
Approving but please remove index.html. (Or better, maybe add .gitignore for this)

**Comment by miketaylr** [comment]
> Approving but please remove index.html. (Or better, maybe add .gitignore for this)

oops, wasn't paying that much attention. 🙈

File: index.bs [hunk 1]
```diff
@@ -509,7 +509,7 @@ The <dfn method for=LockManager>request(|name|, |callback|)</dfn> and
 
 1. If |options| was not passed, then let |options| be a new {{LockOptions}} dictionary with default members.
 1. Let |environment| be [=/this=]'s [=/relevant settings object=].
-1. If |environment|'s [=environment settings object/responsible document=] is not [=Document/fully active=], then return [=a promise rejected with=] a "{{InvalidStateError}}" {{DOMException}}.
+1. If |environment|'s [=relevant global object=]'s [=associated Document=] is not [=Document/fully active=], then return [=a promise rejected with=] a "{{InvalidStateError}}" {{DOMException}}.
```
[the remark this answers is not in the document]
Comment (miketaylr) [reply index.bs:512]: I think both are the same, it just depends which you prefer (I kept it as hanging off `|environment|` just for symmetry, no other reason).

[the remark this answers is not in the document]
Comment (miketaylr) [reply index.bs:512]: (happy to change it if you prefer)

**Pull request merged**